bimmer_connected/api/authentication.py

```diff
@@ -270,14 +274,33 @@ async def _login_china(self):
             encrypted = cipher_rsa.encrypt(self.password.encode())
             pw_encrypted = base64.b64encode(encrypted).decode("UTF-8")
 
-            cipher_aes = AES.new(**get_aes_keys(self.region), mode=AES.MODE_CBC)
-            nonce = f"{self.username}|{datetime.datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S.%fZ')}".encode()
+            captcha_res = await client.post(
+                AUTH_CHINA_CAPTCHA_URL,
+                json={"mobile": self.username},
+            )
+            verify_id = captcha_res.json()["data"]["verifyId"]
+
+            for i in range(1, 13):
+                try:
+                    captcha_check_res = await client.post(
+                        AUTH_CHINA_CAPTCHA_CHECK_URL,
+                        json={"position": 0.74 + i / 100, "verifyId": verify_id},
+                    )
+                    if captcha_check_res.json()["code"] == 200:
```

Comment:
Can we just check against request status code, i.e. `captcha_check_res.status_code == 200`?

Reply:
Now, check if `captcha_check_res.status_code == 201`